init/devices.cpp

```diff
@@ -402,40 +414,96 @@ std::vector<std::string> DeviceHandler::GetBlockDeviceSymlinks(const Uevent& uev
 void DeviceHandler::HandleDevice(const std::string& action, const std::string& devpath,
                                  const std::string& upath, bool block,
                                  int major, int minor, const std::vector<std::string>& links) const {
-    std::vector<std::string> all_links(links);
-    std::string alias_link;
+    std::string aliases_linkpath;
+    std::stringstream slfmt;
 
-    // Add the alias link to the device to the list of links to
-    // create.
-    if (GetDeviceAlias(upath, major, minor, alias_link))
-        all_links.push_back(alias_link);
+    // Create a symlink in /dev/aliases that will point to the
+    // alias.
+    slfmt << "/dev/aliases/" << major << "_" << minor;
```

Comment:
When is this format used? This appears to be new code, yet different from the naming format used in `GetDeviceAlias()`.